src/ServicePulse.Host/vue/src/router/vue-router.d.ts

```diff
@@ -8,7 +8,6 @@ export {};
 
 declare module "vue-router" {
   interface RouteMeta {
-    // is optional
-    title?: string;
+    title: string;
```

Comment:
i haven't looked to see where RouteMeta is used, but would it make sense to have RouteItem extend RouteMeta?

Reply:
I copied this from https://router.vuejs.org/guide/advanced/meta.html#TypeScript and didn't really look much into it.
This is just to make TS happy!